fe2-android/docs/README.md

```diff
@@ -36,7 +37,9 @@ app/src
 │    │                           
 │    ├── res                    # resources (layouts, strings, dimensions, etc.)
 │    │    
-│    └── java/P                 
+│    └── java/P       
+│         ├── di                # module containing the hilt modules used for dependecy injection
```

Comment:
I'm wondering, does it make sense to have all the injected dependencies in one package, or should they be stored in semantic packages ? (e.g., a `utility/camera` package for instance)
What's your opinion on this?

Reply:
It only stores the modules that contain the explicit injection rules. (Opposed to the implicit defined directly on the dependencies with @Inject on the constructor)

Currently, it only has 5 classes.

I might need to be more precise on the description, but I wanted to stay concise with the line length.

Reply:
just to clarify, you're saying it only contains the mappings, and not the injected classes themselves?

Reply:
Yes, that's basically it.

The classes holding those mappings are called Module in the Hilt terminology.